shimtest/mockstub.go

```diff
@@ -400,9 +403,33 @@ func (stub *MockStub) GetCreator() ([]byte, error) {
 	return stub.Creator, nil
 }
 
+// SetTransient to set TransientMap
+func (stub *MockStub) SetTransient(tMap map[string][]byte) error {
+	if stub.signedProposal == nil {
+		return fmt.Errorf("signedProposal is not init")
+	}
+	payload := &pb.ChaincodeProposalPayload{
+		TransientMap: tMap,
+	}
```

Comment:
This variable is not referenced after declaration and can be provided directly to the marshal call.

```
payload, err := proto.Marshal(&pb.ChaincodeProposalPayload{
    TransientMap: tMap,
})
if err != nil {
...
}
```

Same is true of the proposal bytes.